x/aura/types/msgs.go

```diff
@@ -400,3 +401,61 @@ func (msg *MsgRemovePauser) GetSignBytes() []byte {
 func (*MsgRemovePauser) Route() string { return ModuleName }
 
 func (*MsgRemovePauser) Type() string { return "aura/RemovePauser" }
+
+//
+
+var _ legacytx.LegacyMsg = &MsgAddBlockedChannel{}
+
+func (msg *MsgAddBlockedChannel) ValidateBasic() error {
+	if _, err := sdk.AccAddressFromBech32(msg.Signer); err != nil {
+		return fmt.Errorf("invalid signer address (%s): %w", msg.Signer, err)
+	}
+
+	if !channeltypes.IsValidChannelID(msg.Channel) {
+		return fmt.Errorf("invalid channel (%s)", msg.Channel)
+	}
+
+	return nil
+}
+
+func (msg *MsgAddBlockedChannel) GetSigners() []sdk.AccAddress {
+	signer, _ := sdk.AccAddressFromBech32(msg.Signer)
+	return []sdk.AccAddress{signer}
+}
+
+func (msg *MsgAddBlockedChannel) GetSignBytes() []byte {
+	return sdk.MustSortJSON(ModuleCdc.MustMarshalJSON(msg))
+}
+
+func (*MsgAddBlockedChannel) Route() string { return ModuleName }
+
+func (*MsgAddBlockedChannel) Type() string { return "aura/AddBlockedChannel" }
+
+//
+
+var _ legacytx.LegacyMsg = &MsgRemoveBlockedChannel{}
+
+func (msg *MsgRemoveBlockedChannel) ValidateBasic() error {
+	if _, err := sdk.AccAddressFromBech32(msg.Signer); err != nil {
+		return fmt.Errorf("invalid signer address (%s): %w", msg.Signer, err)
+	}
+
+	if !channeltypes.IsValidChannelID(msg.Channel) {
+		return fmt.Errorf("invalid channel (%s)", msg.Channel)
+	}
```

Comment:
This check probably isn't necessary since it is impossible to add an invalid channel. But I guess it doesn't hurt.